src/Components/Common/components/CardV2.tsx

```diff
@@ -0,0 +1,19 @@
+export default function CardV2(props: {
+  children: JSX.Element | JSX.Element[];
+  className?: string;
+  raised?: "sm" | "md" | "lg" | "xl" | "2xl" | "inner";
+  border?: boolean;
+  rounded?: "sm" | "md" | "lg" | "xl" | "2xl" | "3xl";
+}) {
+  return (
+    <div
+      className={`${props.className}  bg-white p-5 ${
+        props.rounded ? `rounded-${props.rounded}` : "rounded-lg"
+      } ${props.raised ? `shadow-${props.raised}` : "shadow-md"} ${
```

Comment:
@GokulramGHV string interpolating tailwind classes isn't nice. tailwind purge will not be able to detect the usages correctly. include full tailwind classnames.